__tests__/Chat.js

```diff
@@ -1,14 +0,0 @@
-import 'react-native';
```

Comment:
This and other tests sporadically fail because of their async methods/functions. They also block other synchronous/faster unit tests from happening. Would advise to move towards testing smallest components.